content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -26,6 +26,10 @@ the Linux man-pages project.
 {{< /note >}}
 ## {{% heading "prerequisites" %}}
 
+{{< note >}}
+`sysctl` is a Linux-specific command-line tool used to configure various kernel parameters
```

Comment:
Per [this comment](https://github.com/kubernetes/website/issues/40201#issuecomment-1479868095), it'd be worthwhile to add that the parameters can be set for a pod running on Windows but will do nothing.